graphql/integration_atomic_test_util.go

```diff
@@ -31,33 +31,110 @@ import (
 	"github.com/evergreen-ci/evergreen/model/commitqueue"
 	"github.com/evergreen-ci/evergreen/model/distro"
 	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/evergreen-ci/evergreen/model/log"
 	"github.com/evergreen-ci/evergreen/model/task"
-	"github.com/evergreen-ci/evergreen/model/tasklog"
 	"github.com/evergreen-ci/evergreen/model/user"
+	"github.com/evergreen-ci/evergreen/taskoutput"
 	"github.com/evergreen-ci/gimlet"
+	"github.com/evergreen-ci/utility"
 	"github.com/mongodb/grip"
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.mongodb.org/mongo-driver/bson"
-	"go.mongodb.org/mongo-driver/mongo"
 )
 
 type AtomicGraphQLState struct {
-	ServerURL   string
-	ApiUser     string
-	ApiKey      string
-	Directory   string
-	TaskLogDB   string
-	TaskLogColl string
-	TestData    map[string]json.RawMessage
-	Settings    *evergreen.Settings
+	ServerURL      string
+	ApiUser        string
+	ApiKey         string
+	Directory      string
+	DBData         map[string]json.RawMessage
+	TaskOutputData map[string]json.RawMessage
+	Settings       *evergreen.Settings
 }
 
 const apiUser = "testuser"
 const apiKey = "testapikey"
 
-func setup(t *testing.T, state *AtomicGraphQLState) {
+func MakeTestsInDirectory(state *AtomicGraphQLState, pathToTests string) func(t *testing.T) {
```

Comment:
Would appreciate a comment here or at `setupTaskOutputData()` describing the role of `task_output_data.json` files

Reply:
Added documentation at the top of the file. Let me know if that suffices!